### ISSUE TITLE
New mpi4py support in Swift/T

### ISSUE THREAD

**Issue by jmjwozniak** [swift-lang/swift-t#153]
Need to mix @par functions with python() functions.

**Comment by jmjwozniak** [comment]
CC @mdorier @mrakitin 
This WIP was merged to master and caused extra debugging output.

**Comment by mrakitin** [comment]
Good to know, thanks @jmjwozniak! Any plans to clean up the debug output in the next release?

**Comment by jmjwozniak** [comment]
Fixed now.

**Comment by mrakitin** [comment]
Great, thanks! So, it's in the master branch now, but not released yet. Correct? If there is a released version, I'd go ahead and create a new conda package with the fix.

**Comment by jmjwozniak** [comment]
Right.  We might be about to release a minor version though for several minor issues...
